.github/workflows/rebase-linux-nvidia-6.12.yml

```diff
@@ -0,0 +1,57 @@
+name: Update and Rebase linux-nvidia-6.12
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  update-rebase-branch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout current branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref_name }}
+          fetch-depth: 0
+
+      - name: Add upstream remote (if not exists)
+        run: |
+          if ! git remote | grep -q upstream; then
+            git remote add upstream https://github.com/gregkh/linux.git
+          fi
+
+      - name: Fetch upstream tags
+        run: git fetch upstream --tags
+
+      - name: Find latest stable tag for branch
+        id: latest_tag
+        run: |
+          tag=$(git tag -l 'v6.12.*' --sort=-v:refname | head -1)
+          echo "tag=$tag" >> $GITHUB_OUTPUT
+
+      - name: Fetch patch branch
+        run: git fetch origin linux-nvidia-6.12
+
+      - name: Find merge base between patch branch and updated LTS
+        id: merge_base
+        run: |
+          base=$(git merge-base origin/linux-nvidia-6.12 ${{ steps.latest_tag.outputs.tag }})
+          echo "base=$base" >> $GITHUB_OUTPUT
+
+      - name: Create new branch for rebased patches
+        run: |
+          git checkout -b linux-nvidia-6.12-${{ steps.latest_tag.outputs.tag }} origin/linux-nvidia-6.12
+
+      - name: Set git user identity
+        run: |
+          git config --local user.name "github-actions[bot]"
+          git config --local user.email "github-actions[bot]@users.noreply.github.com"
+
+      - name: Rebase patch branch onto updated LTS
+        run: |
+          git rebase --onto ${{ steps.latest_tag.outputs.tag }} ${{ steps.merge_base.outputs.base }}
+
+      - name: Push rebased patch branch
+        run: |
+          git push origin HEAD:linux-nvidia-6.12-${{ steps.latest_tag.outputs.tag }}
```

Comment:
We can probably add `--force` here, so if we apply more "SAUCE" patches to our repo it'll update the rebased branch automatically. At the end these are just temporary branches automatically generated, so we shouldn't care about overwriting them.